### DISCUSSION TITLE
printing problems

### DISCUSSION THREAD

**Discussion by Oleksij79** [arkypita/LaserGRBL#2152] [Q&A]
grbl laser is faulty. when printing a picture, it always makes 1-2-3 stripes above half, as when. other programs print without stripes, but the quality of Laser GRBL is better - but I can't solve the problem with those stripes

**Comment by Oleksij79** [comment]
solution error. there were 7 lines in the settings, and 80,000 in the configurations, so you need to set 8 lines

**Reply by Tribo-2dot0** [reply]
Use a calculadora ao lado de onde insere linhas/mm.

**Comment by Ruth678** [comment]
Hi,
I am currently experiencing challenges with butcher paper printing. The paper seems to jam frequently, leading to disruptions and inefficiencies in the printing process. 
Despite adjusting the settings and trying different methods, the issues persist, affecting the quality and timeliness of my print jobs. I am actively seeking solutions to overcome these [butcher paper printing](https://customparchmentpaper.com/butcher-paper/) problems and improve the overall printing experience.
